test/parallel/test-frozen-global.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// This tests that the globals can still be loaded after globalThis is freezed.
+
+require('../common');
+const assert = require('assert');
+
+Object.freeze(globalThis);
+const keys = Reflect.ownKeys(globalThis).filter((k) => typeof k === 'string');
```

Comment:
```suggestion
const keys = Reflect.ownKeys(globalThis);
```

Reply:
Hmm, I don't think having only the keys is an improvement for debugging the failures? At least it helped my figuring out what's going on when I could just `console.log()` the failures. Sometimes I think there is a tendency in our tests that made the tests too simple and DRY so it makes the tests harder to debug, but I'd personal prefer debuggability over simplicity...

Reply:
Did you post your comment on the right thread? This suggestion is removing a filter (so we run the test on all the keys), it's not adding one.

Reply:
Oh, right, the comment should be posted to the for-loop below. But I don't think we need to run the test on all keys though, otherwise we run into the undici dispatch symbol, and I don't see that it's Node.js core that should test the symbol works on a frozen globalThis?

Reply:
Restricting to string keys is a bit weird, I think it would be better to be able to access all global keys when the global object is sealed, and add to one that would fail to the expected failures set. If that's harder than it's worth, we should add a comment explaining that.

Reply:
Personally I'd only care about string keys because it would be weird that contextual lookup on the global stops working once globalThis is sealed. Symbols not so much because one can't do contextual lookup on the global with symbols, and also these symbols aren't necessarily visible to/fixable by Node.js core (like the undici one). But I can add a comment.